src/components/KTabs/KTabs.vue

```diff
@@ -75,6 +78,9 @@ const props = defineProps({
     type: Boolean,
     default: false,
   },
+  /**
+   * @deprecated
```

Comment:
I removed this prop from docs (our standard practice for deprecated props is to remove them from Kongponents docs).

Reply:
> I removed this prop from docs (our standard practice for deprecated props is to remove them from Kongponents docs).

Standard is actually to mark as deprecated 

![image](https://github.com/user-attachments/assets/1bfb7ed0-3414-41e7-8250-4084ee1001ee)

Reply:
Well maybe we can't use a word "standard" here since we have different patterns:
* KModalFullscreen is deprecated and we don't have the component docs page
* KTable is deprecated but because it's such a crucial component internally we still have the page up
* `label` prop in KDropdown is deprecated and is not present in docs
* same applies to `isResizable` and `hasError` props in KTextArea
* in fact, same applies to every deprecated prop in any component

So how do we want to go about this one? In my opinion, we don't want to promote usage of it so we should remove it from docs.